### DISCUSSION TITLE
How Do I Override Settings at Runtime Only For the Current Instance

### DISCUSSION THREAD

**Discussion by VehementHam** [kovidgoyal/kitty#7340] [Q&A] [answered]
I would like to be able to change a setting at runtime, a keybind for example, for the running instance only. I mean the intance in which the command was run from.

**Comment by kovidgoyal** [comment, marked as answer]
See https://sw.kovidgoyal.net/kitty/remote-control/

In particular load-config --override

On Fri, Apr 12, 2024 at 11:35:04AM -0700, VehementHam wrote:
> I would like to be able to change a setting at runtime, a keybind for example, for the running instance only. I mean the intance in which the command was run from.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/7340
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Comment by VehementHam** [comment]
Nice! Thank you for the help.

**Reply by kovidgoyal** [reply]
If you are trying to set keymaps dynamically, you might be better of using conditional maps. See https://sw.kovidgoyal.net/kitty/mapping/
